benchmark/benchmarks.dart

```diff
@@ -4,11 +4,11 @@ import 'package:benchmark_harness/benchmark_harness.dart';
 
 import 'package:r_tree/r_tree.dart';
 
-final int branchFactor = 16;
-final int randomSeed = 3;
-main() {
+const int branchFactor = 16;
+const int randomSeed = 3;
+void main() {
   print('Running benchmarks...');
-  var collector = ScoreCollector();
+  final collector = ScoreCollector();
   InsertBenchmark(collector, totalItems: 100).report();
   InsertBenchmark(collector, totalItems: 1000).report();
   InsertBenchmark(collector, totalItems: 10000).report();
```

Comment:
As of now there's still a difference between doing add with one item vs many items, so we can probably keep the benchmarks around until we change that in a future PR.